packages/.server-id

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this file shouldn't be here. It should be on .gitignore 
it is a part of phone home and I think it in not on .gitignore by mistake